CFC_DataCollector/trip_analysis/kmedoid.py

```diff
@@ -0,0 +1,122 @@
+import random
+import sys
+
+#code based on K_medoid_2.py in CFC_WebApp/main
```

Comment:
If we delete the one in main, then cluster_pipeline won't run. Let me first integrate it so that mine works for any kind of data, and then we can delete the other one. 

I did not make improvements to the actual k-medoids algorithm, but I changed the file so that it works with data that you pass in instead of user data, I added a distance calculator function that works with my featurization scheme, and I added a method that constructs the distance matrix within the file. I will make a unit test, add comments, and improve this file to work in all cases.